en/docs/develop/integration-artifacts/supporting/functions.md

```
@@ -74,55 +72,28 @@ function validateOrderRequest(OrderRequest request) returns string[] {
-}
+By default, all functions are added to a single `functions.bal` file at the project root. For larger projects, you can split functions into additional `.bal` files grouped by domain.
 
-// Private helper — not accessible from outside this file
-function getDiscount(string? couponCode) returns decimal {
-    match couponCode {
-        "SAVE10" => { return 0.10d; }
-        "SAVE20" => { return 0.20d; }
-        _ => { return 0d; }
-    }
-}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)